libs/Helpers.py

```diff
@@ -142,3 +142,21 @@ def parse_file_for_tolerations(self, filename):
                 else:
                     continue
         return tolerations
+
+    @keyword
+    def install_managed_starburst_addon(self, email_address, license, cluster_name):
+        ocm_client = OpenshiftClusterManager()
+        ocm_client.cluster_name = cluster_name
+        ocm_client.notification_email = email_address
+        license_escaped = license.replace('"', '\\"')
+        result = ocm_client.install_managed_starburst_addon(license=license_escaped, exit_on_failure=False)
+        if not result:
+            self.BuiltIn.fail(
+                "Something got wrong while installing Managed Starburst. Check the logs"
+            )
+    
+    @keyword
+    def uninstall_managed_starburst_using_addon_flow(self, cluster_name):
+        ocm_client = OpenshiftClusterManager()
+        ocm_client.cluster_name = cluster_name
+        ocm_client.uninstall_managed_starburst_addon(exit_on_failure=False)
```

Comment:
No output from this call in case uninstallation fails?

Reply:
this function is just a wrapper for the actual function defined in ocm.py to let us using the latter as RF keyword. The check on uninstall is done in `uninstall_addon` function:
```def uninstall_addon(self, addon_name="managed-odh", exit_on_failure=True):
        """Uninstalls addon"""

        addon_state = self.get_addon_state(addon_name)
        if addon_state != "not installed":
            cluster_id = self.get_osd_cluster_id()
            cmd = "ocm delete /api/clusters_mgmt/v1/clusters/{}/addons/" "{}".format(
                cluster_id, addon_name
            )
            log.info("CMD: {}".format(cmd))
            ret = execute_command(cmd)
            if ret is None:
                log.info(
                    "Failed to uninstall {} addon on cluster "
                    "{}".format(addon_name, self.cluster_name)
                )
                if exit_on_failure:
                    sys.exit(1)```

Reply:
I understand that it's a wrapper, but wouldn't it make more sense to also pass along the output of the function that it's wrapping?